encord/objects/classification.py

```diff
@@ -61,14 +61,14 @@ def children(self) -> Sequence[OntologyElement]:
         """
         return self.attributes
 
-    def create_instance(self) -> ClassificationInstance:
+    def create_instance(self, range_only: bool = False) -> ClassificationInstance:
```

Comment:
Happy to call this something else, but felt like this was the best name I could come up with.

Reply:
Some other options:

`use_ranges`
`without_frames`


Or, ask to pass in `data_type` instead, and we decide whether or not they need to use the `range_only` version of the ClassificationInstance.

Reply:
Passing data type sounds like interesting idea, but might become even more cumbersome, so probably just a flag is ok for now. 
We might want to introduce a different way to create instances instead, like `label_row.create_instance(OntologyAttirbute)` or smth, so it would be immediately aware about the inner details. 
But don't think it should be addressed as part of this pr

Reply:
Understood!